### ISSUE TITLE
process hang: Store part of IRIS archive 

### ISSUE THREAD

**Issue by FreddieAkeroyd** [ISISComputingGroup/IBEX#3640]
IRIS suffered a process hang so would have missed logging blocks from around 04:00 to 10:00 on 5th October. We should make a copy of all values in the instrument archive between these times in case they are needed later, we will also need to record the block -> pv mapping used at the time as the instrument archive is the real pv not the block archive. An appropriate SQL select should do it, maybe beam off day is a good time to do this.

**Comment by Tom-Willemsen** [comment]
I have put a copy of their archive in `\\isis\shares\ISIS_Experimental_Controls\FilesMovedFromInstruments\NDXIRIS`. I just did a mysqldump to be sure that we have captured everything that could be needed later.

The block -> pv mapping is stored in the configurations git repository so this is already version controlled and accessible later if we need it. I have double checked that it is working and pushing to the central git server correctly.
